### PR TITLE
[now-python] Fix UnicodeDecodeError for binary response from handler

### LINKED ISSUE #3147: UnicodeDecodeError in python builder
This is essentially the opposite of #3091, there's no way to return anything but UTF-8 (such as a binary file) from the handler class.

The WSGI handler works around this by encoding the output as a base64 string, and then supplying `encoding` in the result dict, but there's no way to do that from the simpler handler class right now.

### PR DESCRIPTION
**Pull request by jacobmischka**
Fixes #3147

### CODE REVIEW

File: packages/now-python/now_init.py [hunk 1]
```diff
@@ -56,13 +56,22 @@ def now_handler(event, context):
...
-            'body': data,
         }
+
+        data = res.read()
+
+        try:
+            return_dict['body'] = data.decode('utf-8')
+        except UnicodeDecodeError:
+            return_dict['body'] = base64.b64encode(data).decode('utf-8')
+            return_dict['encoding'] = 'base64'
```
Comment (styfle) [inline packages/now-python/now_init.py:71]: Please add a test fixture for this new behavior

Comment (jacobmischka) [reply packages/now-python/now_init.py:71]: Done, I believe.

**Comment by jacobmischka** [comment]
~~Hm, it seems the test didn't include the file, and I don't think `includeFiles` is used in the python builder.~~

Oh, it's the probe that's failing to find the image. Hm.

**Review by styfle — APPROVED** [review]
This is beautiful, thanks! 🤩

**Pull request merged**